### PR TITLE
LL-4091 use spendableBalance in swap origin

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-4091

### CODE REVIEW

**Review by juan-cortes — DISMISSED** [review]
We should always use the spendable balance, like we do for the send/receive flows. This whole concept of the "balanceKey" is not necessary and is prone to errors since accounts can be of different types (`TokenAccount`, `ChildAccount` and `Account`) and perhaps not share the same properties.

```const balance =
    account.type !== "ChildAccount" && account.spendableBalance
      ? account.spendableBalance
      : account.balance;
```

As used on the https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/components/SelectAccount.js#L94-L97 `SelectAccount` component would be more than enough

**Comment by machard** [comment]
@juan-cortes you're saying I should add the snippet you posted in `PerCurrencySelectAccount/Option.js` and that it will be fine everywhere ?
i'll edit that way then, thx

**Review by juan-cortes — APPROVED** [review]
Lgtm

**Review by Arnaud97234 — APPROVED** [review]

File: src/renderer/components/PerCurrencySelectAccount/Option.js [hunk 1]
```diff
@@ -71,6 +71,11 @@ function AccountRow({
 }) {
   const palette = useTheme("colors.palette");
 
+  const balance =
+    account.type !== "ChildAccount" && account.spendableBalance
+      ? account.spendableBalance
+      : account.balance;
```
Comment (gre) [inline src/renderer/components/PerCurrencySelectAccount/Option.js:77]: Can use this https://github.com/LedgerHQ/ledger-live-common/blob/master/src/account/helpers.js#L65

Comment (gre) [reply src/renderer/components/PerCurrencySelectAccount/Option.js:77]: But not blocking

**Pull request merged**